.gitignore

```diff
@@ -31,7 +31,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
-.env
```

Comment:
Acabo de notar que esta misma PR borra un `.env.example` que ya estaba. ¿Puedo preguntar por qué? No encontré nada al respecto en la descripción de la PR o en los mensajes de commit.

Reply:
La verdad no me había dado cuenta de que fue removido, pero les comentare a mis compañeros que los vuelvan a integrar, gracias.

Reply:
> Acabo de notar que esta misma PR borra un `.env.example` que ya estaba. ¿Puedo preguntar por qué? No encontré nada al respecto en la descripción de la PR o en los mensajes de commit.

Si, fue borrado por esta razón [Use rails credentials instead of .env](https://github.com/BC-MAR-21-ROR/proyecto-denuncia-ciudadana-proyecto-team02/commit/ef2b5b2a86ffe88f7bea3c58ead1d048027f7ef7#comments)